aspects/aspects.go

```diff
@@ -530,6 +516,69 @@ func schemaTypesStr(types []SchemaType) string {
 	return sb.String()
 }
 
+func getPathsAndValues(storagePath string, suffixParts []string, val interface{}) (map[string]interface{}, error) {
```

Comment:
maybe exporting this in tests having dedicated could be useful?

Reply:
also this probably needs  a doc comment, I wonder if there could be a clearer name as well?  getValuesThroughPaths ?